README.md

```diff
@@ -8,16 +8,16 @@
 
 - **Compatibility**: 7.4.3 and later
 
-- **Applicable**: Content Hub
+- **Applicable**: Submit Content Solution Pack
 
 
 ## Overview
 
 The Submit Content Form Widget is a user-friendly interface designed for community members to share their solutions. 
-The Submit Content Form Widget is part of the larger Submit Your Content Solution pack, which includes associated playbooks for email. This integration streamlines the submission process and ensures efficient handling of submitted solutions.
+
+The Submit Content Form Widget is part of the larger Submit Your Content Solution pack that empowers community members to seamlessly submit their solutions on FortiSOAR’s Content Hub. By launching their FortiSOAR instance and leveraging the solution pack's playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions.
```

Comment:
Need more information on why user should use this widget instead of submitting it via email. We can mention like consistent user experience within FSR only. With predefined steps, forms, etc. makes is very easy to use solution for all our community users.